kotlin/geodesic-operations/README.md

```diff
@@ -0,0 +1,33 @@
+# Geodesic Operations 
+Demonstrates how to use the GeometryEngine to calculate a geodesic path between two points and measure its distance.
+
+![Geodesic Operations Image](GeodesicOperations.png)
```

Comment:
change to standard ![Geodesic Operations App](geodesic-operations.png)

and update screenshot file name accordingly